### PR TITLE
added an HTML file & amended .gitignore

### PR DESCRIPTION
**Pull request by eFunzo**
Just a comment.

### CODE REVIEW

File: funzo-sample-html.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿<!DOCTYPE html>
+
```
Comment (eFunzo) [inline funzo-sample-html.html:2]: This is no right!

**Review by eFunzo — COMMENTED** [review]
Just a final comment!

File: funzo-sample-html.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="utf-8" />
+    <title></title>
+</head>
+<body>
+    Efunzo
```
Comment (kontchou-rovil) [inline funzo-sample-html.html:9]: need to be in a <p> tag

**Review by Yantiomene — APPROVED** [review]
Nice skeleton for a html file

**Pull request merged**